stdlib/jms/src/main/ballerina/jms/queue_receiver.bal

```diff
@@ -141,11 +141,11 @@ function QueueReceiverActions::receiveFrom(Destination destination, int timeoutI
 function validateQueue(Destination destination) {
     if (destination.destinationName == "") {
         string errorMessage = "Destination name cannot be empty";
-        error queueReceiverConfigError = error(errorMessage);
+        error queueReceiverConfigError = error("{ballerina/jms}JMSError", { message: errorMessage });
```

Comment:
Shall define a constant for this?
@final public string JMS_ERROR_CODE = "{ballerina/jms}JMSError";